specification/storage/resource-manager/readme.md

```diff
@@ -26,9 +26,31 @@ These are the global settings for the Storage API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2023-04
+tag: package-2023-05
 ```
 
+### Tag: package-2023-04
+
+These settings apply only when `--tag=package-2023-05` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-05'
+input-file:
+  - Microsoft.Storage/stable/2023-05-01/storageTaskAssignments.json
```

Comment:
Why other swagger file like storage.json are missed from this tag?
This will make the SDK/doc/... generated from the new API version 2023-05-01 only has the storageTaskAssignments API, but not include all original SRP API.